assets/_scss/elements/_typography.scss

```diff
@@ -148,9 +148,13 @@ dfn {
   left: 0;
   margin-left: 8%;
   position: absolute;
-  top: 0; 
+  top: 0;
 }
 
 .usa-image-text {
   margin-top: 0;
-}
\ No newline at end of file
+}
+
+.usa-drop_text {
+  margin-bottom: 0;
```

Comment:
I don't think we need this class with the legend styling from the other PR in #205